src/components/KonnectorItem.jsx

```diff
@@ -61,7 +61,11 @@ class KonnectorItem extends Component {
     return (
       <Link className="item-wrapper" to={`${router.location.pathname}/${slug}`}>
         <header className="item-header">
-          <img className="item-icon" src={getKonnectorIcon(konnector)} />
+          <img
+            className="item-icon"
+            alt={`logo ${name}`}
```

Comment:
I think we should use something like `alt={t('konnector.logo.alt', name)}`. What do you think ?